osCam/core/views.py

```diff
@@ -119,14 +130,21 @@ def rescaleFrame(frame, scale):
 
 @login_required
 def home(request):
-    return render(request, 'core/home.html')
+	
+    return render(request, 'core/home.html', {
+            "update_path_dialog":True,
+            "path":'/usr/media/uploads/',
+            "nextpaths":['videos', 'thumbnails']
+            })
+
+
 
 def gen(camera):
 	while True:
-		frame = camera.get_frame()
+		# frame = camera.get_frame()
+		frame = b'test'
 		yield (b'--frame\r\n'
 				b'Content-Type: image/jpeg\r\n\r\n' + frame + b'\r\n\r\n')
 def feed(request):
-	return StreamingHttpResponse(gen(MotionDetect()),
+	return StreamingHttpResponse(gen(None),
```

Comment:
I'm not sure what is happening here, but I'm pretty sure it's not right. I designed gen to take a MotionDetect object, and you are passing it "None"?

Reply:
was just from me playing around with it, just put it back! glad you caught it

Reply:
if this is resolved, shouldn't I see the change reflected in the code?